### PR TITLE
[BUBR-885] add custom color ranges

### PR DESCRIPTION
**Pull request by taeymma**
* added custom color ranges (as treshold)
* added sample mock geojson
* tested with quark output and belaqi colors

Screenshot 2022-09-19 at 11 38 17

### CODE REVIEW

**Comment by braemJef** [comment]
Hmmmmm should we keep this file? [examples/demo-app/src/mock-geojson.json](https://github.com/imec-int/kepler.gl/pull/8/files#diff-527be6643300a95d7d6fdc391baf8008fe1b09de32982167f2eaee1252140e31)

**Review by braemJef — APPROVED** [review]

File: src/components/common/color-legend.js [hunk 1]
```diff
@@ -91,7 +91,9 @@ const getQuantLegends = (scale, labelFormat) => {
 
   const labels = scale.range().map(d => {
     const invert = scale.invertExtent(d);
-    return `${labelFormat(invert[0])} to ${labelFormat(invert[1])}`;
+    return `${labelFormat(invert[0] ? invert[0] : 0)} to ${
+      invert[1] ? labelFormat(invert[1]) : Infinity
+    }`;
```
Comment (ismakutl) [inline src/components/common/color-legend.js:96]: can we write this in a more readable way? something like this:

```suggestion
    let from = 0;
    let to = Infinity;
    if (labelFormat(invert[0]) {
      from = invert[0];
    }
    if (invert[1]) {
      to = labelFormat(invert[1])
    }
    return `${from} to ${to}`;
```

**Review by ismakutl — CHANGES_REQUESTED** [review]
LGTM!
have 2 other questions/remarks:
- does this also work with the point layer?
- @braemJef is right, we shouldn't include a file that big (~17MB) 😄 maybe we should downsize?

File: src/components/common/color-legend.js (hunk 1 above)
Comment (taeymma) [reply src/components/common/color-legend.js:96]: I quickly tested with a point layer from the examples and it worked, but I didn't pass a range there.

**Review by ismakutl — APPROVED** [review]

**Pull request merged**